rocky/katalogus/views/mixins.py

```diff
@@ -60,6 +60,22 @@ def is_required_field(self, field: str) -> bool:
         return self.plugin_schema and field in self.plugin_schema.get("required", [])
 
 
+class NormalizerMixin(OctopoesView):
```

Comment:
Why does it extend `OctopoesView`? It doesn't seem to depend on it.

It's also unusual to have mixins depending on views, its usually the other way around.

Reply:
I don't know why it's this way around. I copied this from `BoefjeMixin`, but it's indeed not needed for this `NormalizerMixin`